hal/halx86/apic/apicsmp.c

```diff
@@ -84,13 +85,10 @@ ApicRequestGlobalInterrupt(
 
 /* SMP SUPPORT FUNCTIONS ******************************************************/
 
-// Should be called by SMP version of HalRequestIpi
 VOID
-NTAPI
+NTAPI /* HalRequestIpi (CONFIG_SMP) */
```

Comment:
What does this comment mean ?

Reply:
Only called by HalRequestIpi when using the CONFIG_SMP definition. This was just for the sake of organization, I can remove it's a bit of a weird comment. :) just lemme know if you'd like me to take it out

Reply:
Then "Only called by SMP version of HalRequestIpi" would work. Those telegraphic comments don't help to understand.